src/NHibernate/Linq/LinqExtensionMethods.cs

```diff
@@ -2395,26 +2395,6 @@ public static Task<List<TSource>> ToListAsync<TSource>(this IQueryable<TSource>
 
 		#endregion
 
-		public static IQueryable<T> Query<T>(this ISession session)
```

Comment:
NHibernate v5 assembly is binary incompatible with previous version, so just remove these methods. In the majority of cases the change is source compatible and would not cause any problems. The only case it breaks if an user has used `LinqExtensionMethods.Query` as a regular static method, but I expect this to be a really minor case.